pachyderm/templates/postgresql/storageclass.yaml

```diff
@@ -0,0 +1,21 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  labels:
+    app: postgres
+    suite: pachyderm
+  name: postgres-storage-class
+parameters:
+  type: pd-ssd
+provisioner: {{ if eq .Values.pachd.storage.backend "GOOGLE" -}}
+  kubernetes.io/gce-pd
+  {{- else if eq .Values.pachd.storage.backend "AMAZON" -}}
+  kubernetes.io/aws-ebs
+  {{- else if eq .Values.pachd.storage.backend "LOCAL" -}}
+  k8s.io/minikube-hostpath
```

Comment:
I'm ok to make the storage path adjustments we chatted about in the meeting this morning in a subsequent PR, but we should capture in an issue

Reply:
https://github.com/pachyderm/helmchart/issues/87